### ISSUE TITLE
No matching version found for conventional-changelog-preset-loader@^1.1.3

### ISSUE THREAD

**Issue by yvele** [conventional-changelog/conventional-changelog#279]
I'm not using `conventional-changelog` but I get lot of errors when running `npm install` 

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for conventional-changelog-preset-loader@^1.1.3
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'conventional-changelog'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/yves/.npm/_logs/2018-02-13T16_49_03_627Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR!@mycompany/mypackage@1.0.0-beta.0 bootstrap: `npm install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @mycompany/mypackage@1.0.0-beta.0 bootstrap script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/yves/.npm/_logs/2018-02-13T16_49_03_680Z-debug.log
```

@gautierrr CC

![image](https://user-images.githubusercontent.com/4225430/36162957-2d7aaf5c-10e8-11e8-9fa0-28eae75f95e9.png)

**Comment by echoes221** [comment]
Also seeing this same issue - borked our pipeline.

**Comment by Tapppi** [comment]
There is currently a security investigation going on that probably caused this.

/CC @nexdrew @bcoe was this unpublished as a result?

**Comment by mrtnbroder** [comment]
🚨 🚨 🚨

**Comment by Magneticmagnum** [comment]
This also broke for our team during `npm install` with our projects.

**Comment by Darper** [comment]
Ditto

**Comment by yvele** [comment]
As a workaround, is there a way with npm to force a previous version of a package? For example `conventional-changelog-preset-loader@1.1.2`? Maybe by editing `package-lock.json`? But `npm install` is failing before `package-lock.json` has been generated.. 🤔

**Comment by viralpickaxe** [comment]
😠 guys come on...

**Comment by damienpuig** [comment]
🎉 🎉 🎉

**Comment by yvele** [comment]
## Workaround

`npm install --production` 👍

**Comment by jeyglk** [comment]
🔥 🔥 🔥 🔥 🔥 🔥

**Comment by viralpickaxe** [comment]
🔥 🚒

**Comment by christinecha** [comment]
Womp womp.

**Comment by NProtagoras** [comment]
I think the issue here that the package is being used by lerna internally, so most of us can't really use the workaround to fix the issue :)

**Comment by ardeois** [comment]
@bcoe any update on this? Are you aware of the issue?

**Comment by ajeffrey** [comment]
no pressure but we've got 100+ developers hammering F5 on this page.

**Comment by rarkins** [comment]
![image](https://user-images.githubusercontent.com/6311784/36163522-ec1a37a6-10e9-11e8-9c3c-592db8bd000c.png)

**Comment by yvele** [comment]
I personally use lerna [that is using](https://github.com/lerna/lerna/issues/1264) `conventional-changelog-preset-loader` as a dev dependency

https://github.com/lerna/lerna/blob/b28d8dac8ba75f63f8d6c6eea5494fddcfaeceec/package.json#L37-L39

I'm thinking... is there a way for npm to only install root dev dependencies and not deep dev dependencies (dev dependencies of other dependencies) 🤔

**Comment by isnifer** [comment]
@yvele same here - root deps for lerna

**Comment by rarkins** [comment]
While you are refreshing, maybe you can pass the time with [Should you Pin your Javascript Dependencies?](https://renovateapp.com/docs/deep-dives/dependency-pinning)

**Comment by SteveCastle** [comment]
rip lerna. ⚰️

**Comment by chrinor2002** [comment]
https://github.com/conventional-changelog/conventional-changelog/releases/tag/conventional-changelog%401.1.13
conventional-changelog@1.1.13 was when conventional-changelog-preset-loader@1.1.3 was added.
The tag exists: https://github.com/conventional-changelog/conventional-changelog/releases/tag/conventional-changelog-preset-loader%401.1.3
seems like maybe it was removed or possibly the pipeline that deploys that version broke because of conventional-changelog going out before conventional-changelog-preset-loader?

**Comment by pmcalmeida** [comment]
Fix it please

**Comment by viralpickaxe** [comment]
tl;dr

![flowchart](https://i.imgur.com/KHb1S0v.png)

**Comment by abtrumpet** [comment]
Yep, broke us too! Pin NPM dependencies.

**Comment by Darper** [comment]
If somebody could convince Lerna to pin their dependencies that would actually help us right now. Until then, waiting for this to be fixed.

**Comment by Saurabh3333** [comment]
Fix as soon as possible, please.

**Comment by rigor789** [comment]
I'm pretty sure some packages failed to publish to npm, this is unfortunately a known issue that has affected many monorepos recently. Hopefully npm figures out a fix soon, and we'll all be happy again :)

**Comment by chrinor2002** [comment]
@rigor789 if it's a known issue can you link to the ticket?

**Comment by drphelps** [comment]
As a workaround, you can add the following to your package.json, assuming you're using Yarn:
```
"resolutions": {
  "conventional-changelog-preset-loader": "1.1.2"
}
```

**Comment by yvele** [comment]
A possible quick fix may be to remove `latest` npm tag for those versions 

 - `conventional-changelog-angular` 1.6.4
 - `conventional-changelog-core` 2.0.3
- `conventional-recommended-bump` 2.0.3

Or [`npm unpublish`](https://docs.npmjs.com/cli/unpublish)?

Those are the version [used by lerna ](https://github.com/lerna/lerna/blob/b28d8dac8ba75f63f8d6c6eea5494fddcfaeceec/package.json#L37-L39)

**Comment by pmcalmeida** [comment]
The irony is that this package uses lerna, which is broken because lerna needs this package version.  So he can't publish it because travis breaks when trying to publish.
https://travis-ci.org/conventional-changelog/conventional-changelog/jobs/341023923#L442

**Comment by theboolean** [comment]
This happened because of a security issue: `conventional-changelog` package was hacked, and it contained a Monero miner. I reported it to the devs and they unpublished it (and also `conventional-changelog-preset-loader`). They should re-add a safe version tagged with **1.1.3** to fix this issue.

**Comment by ajeffrey** [comment]
that would explain why our npm installs were taking so long.

**Comment by chrinor2002** [comment]
@theboolean is there a issue link that could corroborate that claim?

**Comment by theboolean** [comment]
@chrinor2002 of course: I reported it on their slack: https://yargs.slack.com/archives/C09PUBXB8/p1518530041000529

**Comment by Saurabh3333** [comment]
When is it expected to start functioning normally? Any guesses??

**Comment by rigor789** [comment]
@chrinor2002 I'm not finding an issue for it, but I know that Evan You had issues publishing packages in a monorepo: https://twitter.com/youyuxi/status/961771123636670464
Another case: https://twitter.com/BenLesh/status/961771537622708224

Also confirmed to be a known issue: https://twitter.com/seldo/status/961780556697497600

Tracked here: https://status.npmjs.org/incidents/xn9hdvgxjbq1

**Comment by bcoe** [comment]
Folks, there was a security issue with conventional-changelog, and a publication error with Lerna due to me lacking publication permissions on on of the modules in this ecosystem.

This will be corrected shortly, but open source is made by humans, and thus human is currently in a Lyft.

**Comment by yvele** [comment]
@bcoe thank you for the information and may the force be with you 👍

**Comment by abtrumpet** [comment]
Welcome to the wonderful world of Javascript :)

**Comment by bcoe** [comment]
this issue is now addressed, thank you everyone for your patience; I will be following up in another thread with information about the security issue that occurred.

**Comment by zackarychapple** [comment]
Thank you for reacting so quick to this issue! 

When the retrospective is complete I'd really like to understand it and help share it so other projects can also learn from it and protect themselves.

**Comment by clakech** [comment]
More info with the security issue https://github.com/conventional-changelog/conventional-changelog/issues/282
